openc3/lib/openc3/microservices/microservice.rb

```diff
@@ -192,6 +198,7 @@ def initialize(name, is_plugin: false)
           @logger.error "#{@name} status thread died: #{e.formatted}"
           raise e
         end
+        ThreadManager.instance.register(@microservice_status_thread)
```

Comment:
Why are we running `@microservice_status_thread` and calling `MicroserviceStatusModel.set` here in initialize as well as in the `run` method. Seems like we're double booking it.

Reply:
I don't think anything is double booked.